src/app/home/home.component.ts

```diff
@@ -28,7 +34,29 @@ export class HomeComponent implements OnDestroy {
     }
   }
 
+  ngOnInit() {
+    this.videoQualitySelect = document.querySelector('#video-quality')! as HTMLSelectElement;
+    this.videoCameraSelect = document.querySelector('#video-camera')! as HTMLSelectElement;
+    this.videoQualitySelect.value =
+      localStorage.getItem('defaultQuality') || '480';
+    this.videoCameraSelect.value =
+      localStorage.getItem('defaultCamera') || 'user';
```

Comment:
This isn't imperative style programming isn't really the Angular way.  Can you take a look at using [`mat-select`](https://material.angular.io/components/select/overview)?  I think the examples they have are pretty good.

Reply:
I found that thing but didn't manage to make it work. Now done that way.